source/server/config/network/http_connection_manager.h

```diff
@@ -124,6 +125,7 @@ class HttpConnectionManagerConfig : Logger::Loggable<Logger::Id::config>,
   bool generate_request_id_;
   Http::DateProvider& date_provider_;
   Http::ConnectionManagerListenerStats listener_stats_;
+  bool proxy_100_continue_;
```

Comment:
nit: const (see if other members can be constified)

Reply:
Hahaha.  I added the config after I did my const check.  Will double check the rest in case.